packages/experiment-browser/src/experimentClient.ts

```diff
@@ -116,7 +113,18 @@ export class ExperimentClient implements Client {
         (config?.serverZone?.toLowerCase() === 'eu'
           ? euFlagsServerUrl
           : Defaults.flagsServerUrl),
+      // Force minimum flag config polling interval.
+      flagConfigPollingIntervalMillis:
+        config.flagConfigPollingIntervalMillis < minFlagPollerIntervalMillis
+          ? minFlagPollerIntervalMillis
+          : config.flagConfigPollingIntervalMillis
+          ? config.flagConfigPollingIntervalMillis
+          : Defaults.flagConfigPollingIntervalMillis,
```

Comment:
Maybe add parenthesis? 
Also `A ? A : B` == `A ?? B` if we're just testing for null or undefined.